elixir/apps/web/lib/web/live/sign_in.ex

```diff
@@ -25,7 +25,7 @@ defmodule Web.SignIn do
       {:ok, socket}
     else
       _other ->
-        raise Web.LiveErrors.NotFoundError
+        raise Web.LiveErrors.NotFoundError, report_to_sentry: false
```

Comment:
This is just raising a tuple? Elixir is funny 😂

You only have one usage of this, why not a different exception?

Reply:
Well we'd have to reimplement another `NotFoundError` and its boilerplate - this is less repetitive overall I think